src/main/java/jp/t2v/xanadu/ops/TakeWhileSpliterator.java

```diff
@@ -0,0 +1,34 @@
+package jp.t2v.xanadu.ops;
+
+
+import java.util.Spliterator;
+import java.util.Spliterators;
+import java.util.function.Consumer;
+import java.util.function.Predicate;
+
+class TakeWhileSpliterator<T> extends Spliterators.AbstractSpliterator<T> {
```

Comment:
Agree.
That design can keep `ops` package small even after lots of `Spliterator` implementation added.
The reason I defined `TakeWhileSpliterator` as top level class was to keep `StreamOps` simple.